### PR TITLE
ubuntu.sh: only add gazebo source once on 22.04

### PR DESCRIPTION
**Pull request by MaEtUgR**
### Solved Problem
When looking into https://github.com/PX4/PX4-Autopilot/issues/21130 I realized this duplication in the script.

### Solution
Only add the gazebo binary source once.

### Test coverage
I successfully ran this exact script on a fresh Ubuntu 22.04.1 VM with `sudo apt-get upgrade` before to not run into https://github.com/PX4/PX4-Autopilot/issues/21130.

### CODE REVIEW

**Comment by MaEtUgR** [comment]
I added two more things:
- installing libfuse2 which is not pre-installed anymore on newer Ubuntu versions but is a requirement for Appimages including QGroundcontrol
- sourcing the .profile in which we add the arm compiler such that it's conveniently available without relogin in that terminal. The message at the end stays because other terminals don't yet have it in the path.

**Comment by MaEtUgR** [comment]
I'll move on with this to unblock https://github.com/PX4/PX4-Autopilot/pull/21173
The main question was basically if it's ok to add libfuse2 to the packages and since it's a dependency of the QGroundControl AppImage and is available on all Ubuntu 18.04+ versions I consider it the right thing to do.

File: Tools/setup/ubuntu.sh [hunk 1]
```diff
@@ -84,6 +84,7 @@ sudo DEBIAN_FRONTEND=noninteractive apt-get -y --quiet --no-install-recommends i
 	gdb \
 	git \
 	lcov \
+	libfuse2 \ # QGroundControl AppImage requirement
```
Comment (SeppeG) [inline Tools/setup/ubuntu.sh:87]: The comment is giving an error

Comment (MaEtUgR) [reply Tools/setup/ubuntu.sh:87]: Thank you for the report 🙏  The fix for it is in https://github.com/PX4/PX4-Autopilot/pull/21200 I'd merge it immediately but it's bundled with a compiler upgrade. I think I'll quickly make a separate pr.

Comment (MaEtUgR) [reply Tools/setup/ubuntu.sh:87]: It was fixed in https://github.com/PX4/PX4-Autopilot/pull/21191 sorry for any inconvenience 🙈

**Pull request merged**